### PR TITLE
Simple Calculator

### PR DESCRIPTION
**Pull request by kalez43**
Wrote a small piece of code to practice simple user input and type changes.

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+user_input = input("Type in your yearly salary = ")
+
```
Comment (Bazarovay) [inline calculator.py]: Please head a header mentioning author name and file description

File: calculator.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+user_input = input("Type in your yearly salary = ")
+
+salary = float(user_input)
+
+tax = 0.18
```
Comment (Bazarovay) [inline calculator.py]: make it a function

File: calculator.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+user_input = input("Type in your yearly salary = ")
+
+salary = float(user_input)
+
+tax = 0.18
+
+income = salary - (salary * tax)
+
+print("This is your remaining yearly gross income:")
```
Comment (Bazarovay) [inline calculator.py]: Use a conditional here, is salary is below something then they have majored in economics.

File: calculator.py (hunk 1 above)
Comment (kalez43) [reply calculator.py]: Can I go in and edit the code from github, or do I need to upload a new file with the changes??

Comment (kalez43) [reply calculator.py]: done

File: calculator.py (hunk 3 above)
Comment (kalez43) [reply calculator.py]: done

**Pull request closed without merging**